pkgs/top-level/python-packages.nix

```diff
@@ -22213,12 +22222,15 @@ in modules // {
   };
 
   xcaplib = buildPythonPackage rec {
-    name = "python-xcaplib-${version}";
-    version = "1.1.0";
+    pname = "python-xcaplib";
+    name = "${pname}-${version}";
+    version = "1.2.0";
+    disabled = isPy3k;
```

Comment:
Py3k is not supported for xcaplib, and although the version currently in nixpkgs is built for it this does not work. 
Running ../bin/xcapclient results in "SyntaxError: Missing parentheses in call to 'print')"